TelCo.ColorCoder/ColorPair.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Drawing;
+
+namespace TelCo.ColorCoder
+{
+    public partial class ColorPair
```

Comment:
ColorPair is a Brian Class... and what is the significance of Partial
Who is responsible for Printing the mannual?

Reply:
Reused the the function for different operations and removed partial class.